components/google-cloud/google_cloud_pipeline_components/experimental/gcp_launcher/custom_job.py

```diff
@@ -14,13 +14,13 @@
 
 
 def create_custom_job(
```

Comment:
I feel this code should probably not exist here, especially given that we are adding component SDK code to the experimental folder. 
This might better exist somewhere in https://github.com/kubeflow/pipelines/tree/master/components/google-cloud/container
So at high level 
- Code related to container are in google-cloud/container
- Code related to SDK are in google-cloud/google_cloud_pipeline_components/

WDYT? (And we don't need to make changes in this PR, if any)

Reply:
Good point, I agree current naming / layout is not ideal. I will cover this in a follow up PR.